cli_utils.py

```diff
@@ -0,0 +1,20 @@
+from typing import List
+
+from app.database.models import Conversation
+
+
+def format_conversations(conversations: List[Conversation]):
+    formatted_conversations = []
```

Comment:
We try to use list comprehensions wherever it possible. I think this function will be more clear if written with list comprehension.